frontend/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
In your code you are using some tag that is not correct in terms of user interactions, please review your code. i.e. you use some <div> as button or input.